src/components/panel/panel.js

```diff
@@ -1087,6 +1090,9 @@ MdPanelRef.prototype._createPanel = function() {
 
           self._addStyles();
           self._configureTrapFocus();
+        }, reject)
+        .then(self._$q.when(self._config['scope'].$eval(onDomAdded)), reject)
```

Comment:
Optional: Consider moving this to the attach function after _createPanel is called. It will be called in the same place but if someone is looking in the code it would make more sense in attach.